### ISSUE TITLE
mupx feature request: multiple virtual hosts

### ISSUE THREAD

**Issue by ffxsam** [arunoda/meteor-up-legacy#501]
It would be nice if mupx supported deploying many apps on one server. Each Meteor app would be on a port such as 3000, 3001, 3002, and so on. Nginx would be listening on port 80 and do a `proxy_pass` into the appropriate Node server.

If mupx uses nginx in the docker container, it seems redundant to have nginx on port 80 and proxy_pass into port 3000, and then have another layer of nginx (for SSL?)

**Comment by jamesgibson14** [comment]
1+

**Comment by MomchilGorchev** [comment]
+1

**Comment by eliot84** [comment]
+1

**Comment by miri-am** [comment]
+1

**Comment by TimBroddin** [comment]
+1

**Comment by ernestoaparicio** [comment]
Agreed, how does this differ from https://github.com/arunoda/meteor-up/wiki/Using-Meteor-Up-with-NginX-vhosts?

**Comment by rootedsoftware** [comment]
@ernestoaparicio I believe the difference is that example is using mup and in it docker isn't running nginx. Can anyone else here verify if that is true?

**Comment by simongroll** [comment]
+1
